components/filter/filter-dimension-set.js

```diff
@@ -142,7 +142,10 @@ class FilterDimensionSet extends LitElement {
 				disabled: value.disabled,
 				key: value.key,
 				selected: value.selected,
-				text: value.text
+				text: value.text,
+				type: value.type,
+				rangeNum: value.rangeNum,
```

Comment:
Can we have the consumer specify `<d2l-filter-dimension-set-date-text-value key="lastHour">`? Then the filter dimension set doesn't need to know about these special properties. Or, maybe we can have the item components implement a common method that could be used here, so that this doesn't need to be aware of the different item types.

Reply:
I'd be concerned that if they just used `key` they might not realize that `key` has to be specific values. With implementing the custom range there become more things that need to be passed through, so I think I favour the common method idea.